jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -39,6 +29,14 @@ pipeline {
             defaultValue: 'main',
             trim: true
         )
+        // Must use agent with 1 executor or gradle check will show a lot of java-related errors
+        // The c524xlarge is the instance type that has the least amount of errors during gradle check
+        // https://github.com/opensearch-project/OpenSearch/issues/1975
+        choice(
+            name: 'AGENT_LABEL',
+            description: 'Choose which jenkins agent to run gradle check',
```

Comment:
nit: 'Choose which jenkins agent to run gradle check on'

Reply:
Resolved, thanks.